code/modules/bluespace_anchor/bluespace_anchor.dm

```diff
@@ -0,0 +1,64 @@
+GLOBAL_LIST_EMPTY(active_bluespace_anchors)
+
+/obj/machinery/bluespace_anchor
+	name = "deployed bluespace anchor"
+	desc = "A deployed bluespace anchor, it consumes a large amount of energy in order to stablise bluespace instabilities and prevent teleporation."
+
+	icon = 'icons/obj/device.dmi'
+	icon_state = "memorizer2"
+
+	var/obj/item/stock_parts/cell/power_cell
+	var/range = 8
+	var/power_usage_per_teleport = 3500
+
+/obj/machinery/bluespace_anchor/Initialize(mapload, obj/item/stock_parts/cell/cell)
+	. = ..()
+	//Move the cell
+	insert_cell(cell)
+	GLOB.active_bluespace_anchors += src
+
+/obj/machinery/bluespace_anchor/Destroy()
+	. = ..()
+	GLOB.active_bluespace_anchors -= src
+	//Delete the power cell
+	if(power_cell)
+		QDEL_NULL(power_cell)
```

Comment:
doesn't this (both the global and qdel) need to happen before the `. = ..()`?

Reply:
Yes